common/vpcclient/vpcfilevolume/get_share_profile.go

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2025 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package vpcfilevolume ...
+package vpcfilevolume
+
+import (
+	"time"
+
+	"github.com/IBM/ibmcloud-volume-file-vpc/common/vpcclient/client"
+	"github.com/IBM/ibmcloud-volume-file-vpc/common/vpcclient/models"
+	util "github.com/IBM/ibmcloud-volume-interface/lib/utils"
+	"go.uber.org/zap"
+)
+
+// GetShareProfile GET to /shares/profiles/{profile-name}
+func (vs *FileShareService) GetShareProfile(name string, ctxLogger *zap.Logger) error {
+	ctxLogger.Debug("Entry Backend GetShareProfile")
+	defer ctxLogger.Debug("Exit Backend GetShareProfile")
+
+	defer util.TimeTracker("GetFileShare", time.Now())
```

Comment:
TimeTracker is not aligned with method name

Reply:
Done